HelpSource/Classes/SplayAz.schelp

```diff
@@ -44,7 +45,15 @@ speaker will be directly in front. Should be 0.5 if the front
 bisects a side of the polygon. Then the first speaker will be the
 one left of center.
 argument:: levelComp
-If true, the signal level is adjusted to maintain overall loudness the same (n.reciprocal.sqrt).
+If true, the signal level is adjusted to maintain equal power,
+keeping overall loudness the same (n.reciprocal.sqrt).
+if false, no compensation.
+if float, tunable between 0.0 and 1.0:
+0.0 is no compensation,
+0.5 is equal power,
+1.0 is equal amplitude (1/n)
```

Comment:
As above, consider using the `list::` tag to format these arg descriptions

Reply:
done